### DISCUSSION TITLE
Don't depend on a specific react-select version

### DISCUSSION THREAD

**Discussion by RobinClowers** [csandman/chakra-react-select#317] [Ideas]
It would be great if this library didn't depend on a specific version of react-select. My project already uses the library, so I either end up shipping two versions, or I have to downgrade to the version used in this library.

**Comment by csandman** [comment]
Unfortunately, this package is pretty tightly coupled with `react-select` so it's pretty important that the version is strictly specified, otherwise the custom components it provides have the potential to not be fully compatible (or a few other issues I've dealt with in the past with type dependencies). However, you're not wrong that this package was behind the latest `react-select` version ([`v5.8.0`](https://github.com/JedWatson/react-select/releases/tag/react-select%405.8.0)) for a while now, and that's my bad. Historically, I've been pretty good about keeping them in sync, but recently I've been very busy and this project hasn't received as much attention as I'd like. And the main reason I had put off upgrading to this one was the decent amount of accessibility changes that were implemented that I wanted to make sure were compatible with the custom elements in this package.

I did just push a new version last night though that uses the newest version of `react-select`, [`v4.8.0`](https://github.com/csandman/chakra-react-select/releases/tag/v4.8.0)! So hopefully that at least fixes your issue of needing to manually downgrade the react-select version.

---

On a side note, `react-select` really doesn't receive a ton of new updates these days. `v5.8.0` was released on Nov 6, 2023. So hopefully this issue with `chakra-react-select` depending on a stale version doesn't come up again soon. And if they do release a new version, I'm subscribed to their releases, so I'll try and update this package sooner next time.

**Reply by RobinClowers** [reply]
That all makes sense, thanks you for the detailed response and the new version!

**Reply by csandman** [reply]
No problem, happy to help!
